src/common/assets/base.css

```
@@ -19,6 +19,12 @@
   --vt-c-text-light-2: rgba(60, 60, 60, 0.66);
   --vt-c-text-dark-1: var(--vt-c-white);
   --vt-c-text-dark-2: rgba(235, 235, 235, 0.64);
+
+  --PrimaryColor1: #0D1321;
```

The variable name should be changed to something like
```suggestion
  --primary-gray-dark: #0D1321;
```
for every color, because with `PrimaryColor1`:
- there is no way to figure out which color is actually used
- the keyword `Color` seems redundant
- the current style seems to use `kebab-case`, while you're introducing `PascalCase`